src/main/java/be/seeseemelk/mockbukkit/Coordinate.java

```diff
@@ -5,49 +5,48 @@
  */
 public class Coordinate
 {
-	public int x;
-	public int y;
-	public int z;
+    public int x;
```

Comment:
This is a recurring theme but basically code was done in tabs not spaces which isn't normalized. XML has 2 spaces indent I believe.

Reply:
Oof. I'm personally more of a fan of tabs rather than spaces. I know some people who always set their IDE to render a tab as 2 space, but that makes my eyes bleed. So it's quite nice that we can set it to different settings.
Then again, I'm not sure if I want to start a tabs vs spaces discussion. What do you think @TheBusyBiscuit?

Reply:
There are good arguments for both.

With **spaces** you get universal control of how your code is indented, no matter the platform or settings, which is really handy.
The space character also has the benefit of being easier to use on mobile platforms, albeit not the norm, sometimes I do find myself pushing a quick little fix from my phone.
Spaces simply provide a very huge level of consistency, which is very important for general code style discussions.

With **tabs** however you give every individual the option to set the level of indentation themselves, you hand over this control.
From a memory/data standpoint this will also be slightly more efficient as a tab character is much more lightweight to store than say four individual space characters, even if the compiled result may not care about this. (And easier to read of course since you can just set it to render as two spaces for large files)
From a historical standpoint, the tab character was also designed specifically for indentation, whereas the space character is meant to be used for word seperations.

It really comes down to the preference of the project owner.
I personally prefer spaces, simply as I think they are the most suitable for big open-source projects. They are universal and your code will look the same no matter where you view it, many popular standards like the Google Java Convention have a spaces-only policy too.
But that is just my personal opinion, I don't have anything against following the standard a project owner defines, as there are valid arguments for both variants.

It really doesn't *necessarily* come down to what you choose, as long as it is consistent. Because mixing the two characters creates really bad-looking results most of the time.

Reply:
Let's make it 4 spaces then. Half of the incoming pull requests use it anyway.